interface/forms/eye_mag/css/style.css

```diff
@@ -2971,3 +2981,8 @@ input[type="checkbox"] {
   height: 16px;
   width: 75px;
 }
+
+#BUTTON_DRAW_SDRETINA {
+    margin:8px 0px;
+    width:-webkit-fill-available;
```

Comment:
hi @ophthal, wondering if we need to support firefox as well based on [this post](https://stackoverflow.com/questions/68270468/what-is-the-usage-of-webkit-fill-available#answer-68270606)

Reply:
I must admit I stopped using FireFox when they changed the UI.  Their market share today is pretty low.  I installed the latest version and it appears the UI is back to what is expected, but alas way too late.  Regarding this line, it actually has no effect on the presentation, and the line "width:-webkit-fill-available;" can be removed.